fetch/api/resources/keepalive-iframe.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<script src="/common/utils.js"></script>
+<script>
+const xhr = new XMLHttpRequest();
+window.uuid = token();
+// Use a sync XHR to reset the state.
+xhr.open('GET', `../resources/stash-take.py?key=${uuid}`, false);
```

Comment:
I'm not sure this is useful. If I understand correctly, `uuid` will be different every time this test is run. So there will never be an existing key with value `uuid` and this XHR will do nothing.

Reply:
Sounds reasonable, fixed. Thanks.